stacks/api/bucket/src/crud.ts

```diff
@@ -200,7 +204,16 @@ export async function insertDocument(
     throw new ForbiddenException("ACL rules has rejected this operation.");
   }
 
-  return collection.insertOne(document);
+  return collection.insertOne(document).catch(error => {
```

Comment:
You could return a message like `value of the property .title should unique across all documents.`